keyboards/cirrus40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,105 @@
+        KC_Z,     KC_X,     KC_C,     KC_D,     KC_V,          KC_K,     KC_H,     KC_COMM,  KC_DOT,   KC_UNDS,
+                  QK_LLCK,  BTN_ESC,  NAV_SPC,  NM2_TAB,       FUN_ENT,  SYM_BSP,  NUM_DEL,  QK_LLCK
+    ),
+
+    [kOneHand] = LAYOUT(
+        C(KC_A),  C(KC_W),  A(KC_TAB),C(KC_T),  XXXXXXX,       XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+        WSP_L,    TAB_L,    TAB_R,    WSP_R,    KC_ENT,        XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+        C(KC_Z),  C(KC_X),  C(KC_C),  XXXXXXX,  C(KC_V),       XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+                  _______,  _______,  _______,  _______,       _______,  _______,  _______,  _______
+    ),
+
+    [kNavigation] = LAYOUT(
+        XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,       KC_PGUP,  _______,  _______,  _______,  XXX,
+        KC_LGUI,  KC_LALT,  KC_LCTL,  KC_LSFT,  XXXXXXX,       KC_PGDN,  KC_LEFT,  KC_DOWN,  KC_UP,    KC_RGHT,
+        _______,  _______,  _______,  _______,  XXXXXXX,       _______,  KC_HOME,  _______,  _______,  KC_END,
+                  _______,  _______,  _______,  _______,       KC_ENT,   KC_BSPC,  KC_DEL,   _______
+    ),
+
+    [kNumbers] = LAYOUT(
+        QK_BOOT,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,       XXXXXXX,  KC_7,     KC_8,     KC_9,     XXX,
+        KC_LGUI,  KC_LALT,  KC_LCTL,  KC_LSFT,  XXXXXXX,       S(KC_G),  KC_1,     KC_2,     KC_3,     KC_0,
+        QK_RBT,   XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,       XXXXXXX,  KC_4,     KC_5,     KC_6,     _______,
+                  _______,  _______,  _______,  _______,       KC_ENT,   KC_BSPC,  KC_DEL,   _______
+    ),
+
+    [kFnKeys] = LAYOUT(
+        KC_F10,   KC_F9,    KC_F8,    KC_F7,    XXXXXXX,       XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXX,
+        KC_F11,   KC_F3,    KC_F2,    KC_F1,    XXXXXXX,       XXXXXXX,  KC_LSFT,  KC_LCTL,  KC_LALT,  KC_LGUI,
+        KC_F12,   KC_F6,    KC_F5,    KC_F4,    XXXXXXX,       XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXX,
+                  _______,  KC_ESC,   KC_SPC,   KC_TAB,        _______,  _______,  _______,  _______
+    ),
+
+    [kSymbols] = LAYOUT(
+        KC_QUOT,  KC_LPRN,  KC_RPRN,  KC_SCLN,  KC_PERC,       KC_TILD,  KC_PIPE,  KC_AMPR,  KC_GRV,   KC_DQT,
+        KC_EXLM,  KC_EQL,   KC_SLSH,  KC_PLUS,  KC_HASH,       KC_ASTR,  KC_LSFT,  KC_LCTL,  KC_LALT,  KC_LGUI,
+        KC_CIRC,  KC_LCBR,  KC_RCBR,  KC_DLR,   XXXXXXX,       KC_BSLS,  KC_AT,    KC_LBRC,  KC_RBRC,  KC_QUES,
+                  _______,  KC_ESC,   KC_SPC,   KC_TAB,        _______,  _______,  _______,  _______
+    ),
+};
+
+const char chordal_hold_layout[MATRIX_ROWS][MATRIX_COLS] PROGMEM =
+    LAYOUT(
+        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
+        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
+        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
+             '*', '*', '*', '*', '*', '*', '*', '*'
+    );
```

Comment:
```suggestion
#if defined(CHORDAL_HOLD)
const char chordal_hold_layout[MATRIX_ROWS][MATRIX_COLS] PROGMEM =
    LAYOUT(
        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
        'L', 'L', 'L', 'L', 'L', 'R', 'R', 'R', 'R', 'R',
             '*', '*', '*', '*', '*', '*', '*', '*'
    );
#endif
```

Reply:
It should just be removed, as it's exactly as what would be generated from the keyboard.json.